src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const a = document.querySelector('#largeImg');
+const img = document.querySelector('#thumbs');
+
+img.addEventListener('click', (eve) => {
+  eve.preventDefault();
+  eve.target.closest('.list-item__link');
```

Comment:
This line is redundant because the result of `eve.target.closest('.list-item__link')` is not being stored or used. You can remove this line to avoid confusion.